src/xrEngine/xr_input.cpp

```diff
@@ -195,7 +195,7 @@ pcstr KeyToMouseButtonName(const int dik)
 bool CInput::get_dik_name(int dik, LPSTR dest_str, int dest_sz)
 {
     xr_string keyname;
-    static std::locale locale("");
+    static std::locale locale("ru_RU.UTF-8");
```

Comment:
Just NO! The game is a world-wide product localized to several languages. We should not use hardcoded locales in the code.

Reply:
Need just add getting locale from environment variable

Reply:
Calling std::locale locale(""); - this is set default locale, while only very new libc set defaul locale without this hack.